### PR TITLE
Add 31 wp and 14 links fix and one not working

### PR DESCRIPTION
**Pull request by RustamSultanov**
100% checked https://icotracker.net/upcoming

### CODE REVIEW

File: Whitepapers.md [hunk 1]
```diff
@@ -30,6 +30,7 @@
 | Adperks | AdPerks - Decentralized Digital Marketing | https://adperks.io/wp-content/uploads/2017/09/AdPerks-WhitePaper.pdf | https://adperks.io/ | N/P* |
 | Adpump | BLOCKCHAIN AFFILIATE MARKETPLACE | https://adpump.io/static/media/whitepaper.pdf | https://adpump.io/ | N/P* |
 | Adshares | Enterprise Service Chain | https://adshares.net/assets/docs/ESC.pdf | https://adshares.net/ | N/P* |
+| Adt Finance | Financial Markets Investment Fund (SCAM ALERT!) | https://adtfinance.fr/whitepaper.pdf | https://adtfinance.fr/ | N/P* |
```
Comment (Har01d) [inline Whitepapers.md]: The title doesn’t look good :)

File: Whitepapers.md [hunk 2]
```diff
@@ -564,14 +575,15 @@
...
 | Gladius | Gladius - Blockchain-driven cyber protection network | https://gladius.io/pdf/gladius-whitepaper.pdf | https://gladius.io/ | N/P* |
 | Global Academy Place | Global Academy Place | https://globalacademyplace.com/wp-content/uploads/2017/09/White-paper-GAP.pdf | https://globalacademyplace.com/ico/ | N/P* |
 | Global Crypto Bank | Global Crypto Bank | http://globalbank.io/docs/WP_v5.3.pdf | http://globalbank.io/ | N/P* |
 | Global Jobcoin | Global Jobcoin - Global Crypto Currency for Jobs - Globaljobcoin.com | https://www.globaljobcoin.com/media/files/documents/GJC_Whitepaper.pdf | https://www.globaljobcoin.com/ | N/P* |
 | Global Metrology Project | GLOBAL METROLOGY PROJECT - Initial Coin Offering | https://gmp.im/images/docs/GMP_Project_WhitePapper_EN.pdf | https://gmp.im/ | N/P* |
 | Globetrotter Ecosystem | GLOBETROTTER CRYPTOCURRENCY | https://docs.wixstatic.com/ugd/e8cc17_efa02d0e8c714bbc8911e29a312deb00.pdf | https://www.globetrottercoin.com/ | N/P* |
-| Gluon | GLU Token | https://www.glutoken.com/wp-content/uploads/2017/08/Gluon-Whitepaper.pdf | https://www.glutoken.com/ | N/P* |
```
Comment (Har01d) [inline Whitepapers.md]: We should move that to Skip.md

File: Whitepapers.md [hunk 3]
```diff
@@ -743,7 +756,7 @@
 | Macro | MACRO THE CRYPTOCURRENCY | http://whatismacro.com/Macro Whitepaper - Google Dokumenti.pdf | http://whatismacro.com/ | N/P* |
 | MAD Network | Mad Network | https://madnetwork.io/MAD-Product-Deck.pdf | https://madnetwork.io/ | N/P* |
 | Maecenas | The Decentralized Artgallery | https://www.maecenas.co/Maecenas-WhitePaper.pdf | https://www.maecenas.co/ | |
-| Magnumlink | MAGNUMLINK | https://www.magnumlink.com/WP_ENG_1_booklet.pdf | https://www.magnumlink.com/ | N/P* |
+| Magnumlink | Blockchain platform for easy and convenient trading |https://www.magnumlink.com/WP.pdf | https://www.magnumlink.com/ | N/P* |
```
Comment (Har01d) [inline Whitepapers.md]: Missing space

File: Whitepapers.md [hunk 4]
```diff
@@ -1045,22 +1063,23 @@
 | Share Estate | SHARE ESTATE - Cryptocurrency Backed By Real Estate | http://sharestate.io/wp-content/uploads/2017/09/ShareEstate_eng.pdf | https://sharestate.io/ | N/P* |
 | Shipchain | ShipChain | http://www.shipchain.io/shipchain-whitepaper.pdf | http://www.shipchain.io/ | N/P* |
 | Siacoin | Sia: Simple Decentralized Storage | https://www.sia.tech/whitepaper.pdf | http://sia.tech/ | David Vorick, Luke Champine |
-| Signals | Signals / Build, Train, & Monetise Cryptotrading Strategies | https://cdn.signals.network/docs/Signals-Whitepaper.pdf | https://signals.network/ | N/P* |
+| Sigame | Online trading platform, based on the blockchain technology and smart contracts | https://en.sigame.io/images/White_book_SigSiGame_ICO_v._1.05_en.pdf | https://en.sigame.io/ | N/P* |
+| Signals |  Build, Train, & Monetise Cryptotrading Strategies | https://cdn.signals.network/docs/Signals-Whitepaper.pdf | https://signals.network/ | N/P* |
```
Comment (Har01d) [inline Whitepapers.md]: Extra space

**Comment by Har01d** [comment]
Please review the comments.

File: Whitepapers.md (hunk 2 above)
Comment (RustamSultanov) [reply Whitepapers.md]: This move in Skip when I delete in Whitepaper

File: Whitepapers.md (hunk 1 above)
Comment (RustamSultanov) [reply Whitepapers.md]: Ok I see, I paste this from ico site

**Comment by Har01d** [comment]
Merged, thank you!

**Pull request merged**